src/doc/install_pyenv.md

```diff
@@ -0,0 +1,39 @@
+# Install Silkaj on a Python environment
+
+## Install Pyenv
+
+### Retrieve pyenv sources
+```bash
+git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+```
+
+### Handle shell modifications: point 2,3 and 4.
+- [Follow pyenv install documentation](https://github.com/pyenv/pyenv#installation)
+
+
+### Install latest Python version and create pyenv
+```bash
+pyenv install 3.6.0
+pyenv shell 3.6.0
+pyenv virtualenv silkaj-env
+```
+
+## Install Silkaj
+
+### Retrieve silkaj sources
+```bash
+git clone https://github.com/duniter/silkaj.git
+cd silkaj
+```
+
+### Install dependencies and store them on pyenv environement
+```bash
+pip install -r requirements.txt --upgrade
```

Comment:
In my environment, 'pip' installs for python2. The install for python3 (which is used by silkaj) works with pip3. I didn't use a virtualenv, though, so that may work (not sure about that, I never used a virtualenv so I'll trust you on that one).